deploy_pgcluster.yml

```
@@ -1,13 +1,12 @@
 ---
-
-- name: Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
+- name: deploy_pgcluster.yml | Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
```

```suggestion
- name: Deploy PostgreSQL HA Cluster (based on "Patroni" and "{{ dcs_type }}")
```
why is the playbook name added here?

---

I have incorporated the playbook names into the task names to enhance readability in the logs. However, it's important to note that this change is a proactive step towards our planned migration to collection mode